wiki/Help_Centre/Account_Restrictions/vi.md

```diff
@@ -74,7 +74,7 @@ Thêm vào đó, trong thời hạn một năm, chúng tôi sẽ không chấp n
 
 Với một vài trường hợp, người chơi trở lại sẽ bị cấm tham gia giải đấu chính thức và giải có badge, bất kể dưới vai trò là người chơi hay người điều hành giải. Thời hạn cấm là 1 năm ứng với mỗi lần vi phạm (nếu người chơi quay lại sau lần khiếu nại thứ hai, thì sẽ bị cấm 2 năm). Điều này sẽ được ghi rõ trong email phản hồi khiếu nại.
```

Comment:
Should this be "trong một vài trường hợp"?

Reply:
I think in this case, `Đối với` would sound better. Although `In` literally means `Trong`.
But yeah, I kinda shorten them at the wrong place.

Reply:
`Đối với` is also appropriate in this case IMO. Your choice.